policy_packs/aws/ssm/enforce_install_packages_on_instances_using_ssm_package_installer/default.tfvars.example

```diff
@@ -0,0 +1,5 @@
+# Turbot Resource ARN for the AWS Region where SSM Documents will be placed. For example: 'arn:aws::us-east-1:123456789012'
+region_resource = "<ssm_document_role_arn>"
```

Comment:
Is the example correct? It doesn't seem to match up with the variable name

Reply:
Fixed. Using `region_arn` and `ssm_document_role_arn` for variables now.